src/Network/Network/LoadBalancer/RemoveAzureLoadBalancerBackendPool.cs

```diff
@@ -0,0 +1,121 @@
+
+        [Parameter(Mandatory = true, HelpMessage = "The resource group name of the load balancer.", ParameterSetName = DeleteByNameParameterSet)]
+        [ResourceGroupCompleter]
+        [ValidateNotNullOrEmpty]
+        public string ResourceGroupName { get; set; }
+
+        [Parameter(Mandatory = true, HelpMessage = "The name of the load balancer.", ParameterSetName = DeleteByNameParameterSet)]
+        [Parameter(Mandatory = true, HelpMessage = "The name of the load balancer.", ParameterSetName = DeleteByParentObjectParameterSet)]
+        [ValidateNotNullOrEmpty]
+        public string Name { get; set; }
+
+        [Parameter(Mandatory = false,HelpMessage = "The name of the load balancer.")]
+        [ValidateNotNullOrEmpty]
+        public string LoadBalancerName { get; set; }
+
+        [Parameter(Mandatory = true, HelpMessage = "The load balancer resource.", ValueFromPipeline = true,ParameterSetName = DeleteByParentObjectParameterSet)]
+        public PSLoadBalancer LoadBalancer { get; set; }
+
+        [Parameter(Mandatory = false,HelpMessage = "The backend address pool to remove", ParameterSetName = DeleteByInputObjectParameterSet)]
+        [ValidateNotNullOrEmpty]
+        public PSBackendAddressPool InputObject { get; set; }
+
+        [Parameter(Mandatory = true,ValueFromPipelineByPropertyName = true,ParameterSetName = DeleteByResourceIdParameterSet)]
+        [ValidateNotNullOrEmpty]
+        public string ResourceId { get; set; }
+
+        [Parameter(
+           Mandatory = false,
+           HelpMessage = "Do not ask for confirmation.")]
+        public SwitchParameter Force { get; set; }
+
+        [Parameter(Mandatory = false)]
+        public SwitchParameter PassThru { get; set; }
+
+        public override void Execute()
+        {
+            base.Execute();
+
+            if (this.IsParameterBound(c => c.LoadBalancer))
+            {
+                this.ResourceGroupName = this.LoadBalancer.ResourceGroupName;
+                this.LoadBalancerName = this.LoadBalancer.Name;
+            }
+
+            if (this.IsParameterBound(c => c.InputObject) || this.IsParameterBound(p => p.ResourceId))
+            {
+                var id = this.IsParameterBound(c => c.InputObject) ? this.InputObject.Id : this.ResourceId;
+
+                ResourceIdentifier resourceIdentifier = new ResourceIdentifier(id);
+                this.ResourceGroupName = resourceIdentifier.ResourceGroupName;
+                this.LoadBalancerName = resourceIdentifier.ParentResource.Split('/')[1];
+                this.Name = resourceIdentifier.ResourceName;
+            }
+
+            // check if resource already exists
+            var found = NetworkBaseCmdlet.IsResourcePresent(
+                () => this.NetworkClient.NetworkManagementClient.LoadBalancerBackendAddressPools.Get(
+                this.ResourceGroupName,
+                this.LoadBalancerName, 
+                this.Name));
+
+            if (!found)
```

Comment:
what about not checking and send the delete? If not there I guess deletion is also a success without having to delete anything

Reply:
If we don't check, the API returns a 204 and user thinks it removed something when it didn't, so I think we need to keep this.

Reply:
Sure it could of been deleted, if its deleted we will prompt the user with a resource not found message. That's what this accomplishes. Given the API returns a 204 when delete is called on a non existent object, the alternative suggested will not prompt the user. I don't even really see why that is viable option.